pom.xml

```diff
@@ -53,8 +53,8 @@
     <jackson.version>2.9.5</jackson.version>
     <jackson.databind.version>2.9.5</jackson.databind.version>
     <zookeeper.version>3.4.12</zookeeper.version>
-    <mapr.release.version>5.2.1-mapr</mapr.release.version>
-    <ojai.version>1.1</ojai.version>
+    <mapr.release.version>6.0.1-mapr</mapr.release.version>
```

Comment:
Use 6.1.0-mapr version
It will be introduced after merging this PR:
https://github.com/apache/drill/pull/1489/files#diff-600376dffeb79835ede4a0b285078036R56

Reply:
For this version update, I will let the the PR you referenced [1] handle it.  There are functional test failures with that PR that need to be addressed. 

[1] https://github.com/apache/drill/pull/1489